src/content/docs/logs/forward-logs/overview of log-forwarding.mdx

```diff
@@ -0,0 +1,122 @@
+        * [Syslog/TCP](/docs/logs/log-management/log-api/use-tcp-endpoint-forward-logs-new-relic)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Cloud environments
+      </td>
+      <td>
+       * Amazon AWS:
+        * [CloudFront plugin](/docs/logs/forward-logs/cloudfront-web-logs)
+        * [CloudWatch plugin](/docs/logs/forward-logs/aws-lambda-sending-cloudwatch-logs/)
+        * [FireLens plugin](/docs/logs/forward-logs/aws-firelens-plugin-log-forwarding/)
+        * [Kinesis Firehose](/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose/)
+        * [AWS Lambda for sending logs from S3](/docs/logs/forward-logs/aws-lambda-sending-logs-s3/)
+       * [Google Cloud platform](/docs/logs/forward-logs/google-cloud-platform-log-forwarding)
+       * [Microsoft Azure](/docs/logs/forward-logs/azure-log-forwarding/)
+       * [Heroku](/docs/logs/forward-logs/heroku-log-forwarding/)
+       * [Cloudflare](/docs/logs/forward-logs/cloudflare-logpush-forwarding)
+      </td>
+    </tr>
+    
+    <tr>
+      <td>
+        Docker/Kubernetes containers
+      </td>
+      <td>
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
+        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory. For example:
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).
+      </td>
+    </tr>
+    
+    <tr>
+      <td>
+        Directly from your apps, regardless of where they are hosted
+      </td>
+      <td>
+        * Our <InlinePopover type="apm" /> agent and infrastructure monitoring agent
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Other supported log forwarders
+      </td>
+      <td>
+        * [Our log API](/docs/logs/log-api/introduction-log-api/)
+        * Akamai DataStream 2: 
+          * [Akamai instructions](https://techdocs.akamai.com/datastream2/docs/stream-new-relic)
+          * [Quickstart](https://newrelic.com/instant-observability/datastream2/cb995039-39d8-44d9-97c3-58174a1337f2)
+        * Fastly: 
+          * [Fastly instructions](https://docs.fastly.com/en/guides/log-streaming-newrelic-logs)
+          * [Quickstart](https://newrelic.com/instant-observability/fastly-cdn/c5c5dd30-dcdf-46b6-9412-f9a1bba5a600)
+        * [Vector plugin](/docs/logs/forward-logs/vector-output-sink-log-forwarding)
+        * Use the OpenTelemetry SDK to send logs from your apps to an OpenTelemetry collector, which can forward them to New Relic via OTLP.
```

Comment:
@lchapman4 @marcelschlapfer I'd like to add a docs link with instructions to set this up, but I'm not sure which page for specifically forwarding logs. Can you help me sourcing the best docs link?

Reply:
Maybe use the link to our infra forwarder? https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/

Reply:
But this bullet point is for OpenTelemetry, and we don't have any OTEL info on https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/ :(